include/nifty/graph/undirected_long_range_grid_graph.hxx

```diff
@@ -10,6 +10,9 @@
 
 #include <cstdlib>
 
+#include <pybind11/pybind11.h>
+namespace py = pybind11;
```

Comment:
see previous comments.

Reply:
I already updated GASP and all my tools so that `undirectedLongRangeGridGraph` is no longer needed, so if you want we can remove it right away and test that everything works. 

This was an old modification that I forgot to exclude from the pull request. I will exclude these changes.

Reply:
I removed these changes and also moved some functions out of the `long_range_adjacency` module